lib/cloudkeeper/one/core_connector.rb

```diff
@@ -97,7 +97,7 @@ def handle_errors(call)
       end
 
       def finalize_return_value(return_value, default_return_value, use_return_value)
-        return_value = use_return_value ? return_value : default_return_value
+        return_value = use_return_value && return_value ? return_value : default_return_value
```

Comment:
Oh, jeeez. This could be simplified, right? Or is there something I'm not seeing?
```ruby
return return_value if return_value && use_return_value
default_return_value
```
Btw. is there any way you could use words `return` and `value` more often in this method? :-D

Reply:
Sure, I'll simplify it.